### PR TITLE
Vet_clinic Database: Add Join Table

### PR DESCRIPTION
**Pull request by amare1990**
- [ ] Created two join tables (specializations, and visits).
- [ ] Populated the two join tables with the given data.
- [ ] Wrote SQL queries to answer questions from the project requirement.
- [ ] Took screenshots to verify if the sql query works.
- [ ] Updated the README.md template to describe the task done in the fourth milestone of the project.

**ScreenShots**

Screenshot from 2022-12-15 11-53-18
Screenshot from 2022-12-15 11-22-23
Screenshot from 2022-12-15 10-50-02
Screenshot from 2022-12-15 10-23-30
Screenshot from 2022-12-15 02-23-00
Screenshot from 2022-12-15 02-13-44
Screenshot from 2022-12-15 02-01-59
Screenshot from 2022-12-15 01-36-33

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -42,4 +42,18 @@ FOREIGN KEY (owner_id)
 REFERENCES owners (id) ON DELETE CASCADE;
```
Comment (1GORDON) [inline schema.sql:43]: Great job @amare1990 🤝

- [ ] As stated in the project requirements, kindly add the commands for creating a table named vets.

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 # Vet Clinic Database
```
Comment (1GORDON) [inline README.md:3]: Great job @amare1990 🤝

- [ ] You may as well use the correct readme file. Kindly use this [readme template](https://github.com/microverseinc/readme-template) and customize it accordingly.

File: schema.sql [hunk 3]
```diff
@@ -42,4 +42,18 @@ FOREIGN KEY (owner_id)
...
 
 
+/* Fourth milestone-3rd day */
+CREATE TABLE specializations (vet_id INT NOT NULL REFERENCES vets(id), species_id INT NOT NULL REFERENCES species (id));
+
+ALTER TABLE animals
+ADD CONSTRAINT ANIMALS_PKEY
+PRIMARY KEY (id);
+
+CREATE TABLE visits (animal_id INT NOT NULL REFERENCES animals (id), vet_id INT NOT NULL REFERENCES vets(id));
```
Comment (1GORDON) [inline schema.sql:56]: Great job @amare1990 🤝

- [ ] Kindly add the `visit_date` column to your `visits` table.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @amare1990 ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Descriptive Pull request ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (amare1990) [reply README.md:3]: Hi  @1GORDON.The project clearly requires us to use the template I used. I tried to use the template you suggested for me but it did not give me sense and resorted to using the template from this project requirement. There is no clear space to write DML and Data creation sql statements in the template you suggested for me.

File: schema.sql (hunk 3 above)
Comment (amare1990) [reply schema.sql:56]: I have already used this statement `ALTER TABLE visits
ADD COLUMN date_of_visit date;`

**Review by m05e5 — APPROVED** [review]
Hi @amare1990 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
goodjob

## Highlights
✔️ Descriptive PR 

## Optional suggestions


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**m05e5**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**